.github/workflows/check-build-test.yml

```
@@ -72,9 +72,13 @@ jobs:
       - name: Cache Coursier cache
         uses: coursier/cache-action@v6
 
-      - name: Compile all code with fatal warnings for Java 11 and Scala 2.12/2.13
+      - name: Compile all code with fatal warnings for Java 11 and Scala 2.12
         # Run locally with: env CI=true sbt 'clean ; Test/compile ; It/compile'
-        run: sbt "; +Test/compile; +It/compile"
+        run: sbt ++2.12.17 " ; Test/compile; It/compile"
```

You can use just `++2.12` so you don't need to keep the minor version in sync manually:

```suggestion
        run: sbt ++2.12 " ; Test/compile; It/compile"
```

---

thanks - changed